web/src/message_list_tooltips.js

```diff
@@ -238,12 +238,16 @@ export function initialize() {
         delay: LONG_HOVER_DELAY,
         onShow(instance) {
             const $elem = $(instance.reference);
+            if ($elem.hasClass("no-tooltip")) {
```

Comment:
This has a bad smell to me. This is reworking tooltip logic for a particular, non-obvious case, and I think it's absolutely arguable that either now or in the future, the follow icon would need to take a tooltip.

Reply:
Timabott has suggested in [CZO](https://chat.zulip.org/#narrow/stream/101-design/topic/followed.20topic.20left.20sidebar/near/1745789), not to use tooltip here.

Reply:
That's fine, but there must be a better way to go about addressing this case.